lib/src/local_working_copy.rs

```diff
@@ -1152,6 +1145,7 @@ impl FileSnapshotter<'_> {
                         });
                     }
                     if let Some(new_file_state) = file_state(&metadata) {
+                        present_entries.files.insert(name);
```

Comment:
question: Shouldn't this be inside `process_present_file`? If not, can you add a comment explaining why?

Reply:
It shouldn't because `visit_tracked_files()` doesn't need a set of present files to detect deletion.